ui/src/components/settings/PageStorage.svelte

```diff
@@ -130,6 +130,12 @@
                                 <input type="text" id={uniqueId} required bind:value={s3.endpoint} />
                             </Field>
                         </div>
+                        <div class="col-lg-12">
+                            <Field class="form-field form-field-toggle" name="s3.forcePathStyle" let:uniqueId>
+                                <input type="checkbox" id={uniqueId} required bind:checked={s3.forcePathStyle} />
+                                <label for={uniqueId}>Force Path Style?</label>
```

Comment:
Maybe also remove question mark here?

Note that I'm not a maintainer here, so up to @ganigeorgiev to decide :-)

But I think that usually options don't have question marks - see, for example, "Use S3 storage" option on your screenshot.

Reply:
Updated per @ganigeorgiev's phrasing of it :D I'm not much of a ux person, I'm used to being told where to put stuff!